install.sh

```diff
@@ -60,20 +60,6 @@ else
   tput setaf 2; echo "Docker installed!!!"
 fi
 
-echo " "
-tput setaf 4;
-echo "#########################################################################"
-echo "Installing docker-compose"
-echo "#########################################################################"
-if [ -x "$(command -v docker-compose)" ]; then
-  tput setaf 2; echo "docker-compose already installed, skipping."
-else
-  curl -L "https://github.com/docker/compose/releases/download/v2.23.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-  chmod +x /usr/local/bin/docker-compose
-  ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
-  tput setaf 2; echo "docker-compose installed!!!"
-fi
-
```

Comment:
Why has this been removed actually? Is it because Docker Compose comes with Docker nowadays?

Reply:
the other way round. Docker comes automatically with docker  compose (plugin) and docker-compose (the binary) is not needed anymore.